client/state/wordads/status/schema.js

```diff
@@ -0,0 +1,19 @@
+export const wordadsUnsafeValues = [ false, 'mature', 'private', 'spam', 'other' ];
+export const wordadsStatusSchema = {
+	type: 'object',
+	patternProperties: {
+		'^\\d+$': {
+			type: 'object',
+			required: [ 'ID' ],
```

Comment:
It looks like we're not storing ID, so this schema will also mark all entries as invalid.
<img width="218" alt="screen shot 2016-08-12 at 12 47 40 pm" src="https://cloud.githubusercontent.com/assets/1270189/17635235/1a7eef80-608b-11e6-96f5-45af27536225.png">